test/inductor/test_torchinductor.py

```diff
@@ -3626,7 +3630,7 @@ def fn(*args):
 
     def test_move_arange(self):
         def fn(x):
-            return torch.arange(len(x), device="cpu").to(x.device) + x
+            return torch.arange(len(x), device=self.device).to(x.device) + x
```

Comment:
I think this one is intentionally testing multiple devices `there will be more than one kernel generated`